supervisor/api/addons.py

```diff
@@ -161,8 +162,12 @@ async def list(self, request: web.Request) -> Dict[str, Any]:
                 ATTR_DESCRIPTON: addon.description,
                 ATTR_ADVANCED: addon.advanced,
                 ATTR_STAGE: addon.stage,
-                ATTR_VERSION: addon.latest_version,
-                ATTR_INSTALLED: addon.version if addon.is_installed else None,
+                ATTR_VERSION: addon.version if addon.is_installed else None,
+                ATTR_VERSION_LATEST: addon.latest_version,
+                ATTR_UPDATE_AVAILABLE: addon.need_update
+                if addon.is_installed
+                else None,
```

Comment:
```suggestion
                else False,
```
When it's not installed, there are no updates, if this is None, we need to check if installed in the UI before we check this